renovate.json

```diff
@@ -7,4 +7,4 @@
     "changelog:dependencies"
   ],
   "suppressNotifications": ["prEditedNotification"]
-}
+}
```

Comment:
please tune your IDE/editor to not remove final new-line

Reply:
Okay.